agents/trainer/task.py

```diff
@@ -243,23 +240,22 @@ def gcs_upload(local_dir, gcs_out_dir):
     blob.upload_from_filename(local_file_path)
 
 
-def main(unused_argv):
+def main():
   """Run training."""
   tf.logging.set_verbosity(tf.logging.INFO)
 
   if FLAGS.debug:
     tf.logging.set_verbosity(tf.logging.DEBUG)
 
-  run_config = tf.contrib.learn.RunConfig()
```

Comment:
This can be removed if you have the inclination since the config is not being used and calling it doesn't have any side effect.